python/setup.py

```diff
@@ -53,7 +53,7 @@
     'scipy',
     'scikit-learn',
     'packaging',
-    'interpret-core[required, ebm]==0.1.18',
+    'interpret-core[required, ebm]==0.1.19',
```

Comment:
we should be able to remove the ebm extra again now that the joblib dependency has been added to required!